fly/integration/pipeline_overview_test.go

```diff
@@ -0,0 +1,154 @@
+package integration_test
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/fly/ui"
+	"github.com/fatih/color"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	"github.com/onsi/gomega/ghttp"
+)
+
+var _ = Describe("Fly CLI", func() {
+	Describe("pipeline-overview", func() {
+		var (
+			flyCmd     *exec.Cmd
+			sampleJobs []atc.Job
+		)
+
+		Context("when not specifying a pipeline name", func() {
+			It("fails and says you should give a pipeline name", func() {
+				flyCmd := exec.Command(flyPath, "-t", targetName, "jobs")
+
+				sess, err := gexec.Start(flyCmd, GinkgoWriter, GinkgoWriter)
+				Expect(err).NotTo(HaveOccurred())
+
+				<-sess.Exited
+				Expect(sess.ExitCode()).To(Equal(1))
+
+				Expect(sess.Err).To(gbytes.Say("error: the required flag `" + osFlag("p", "pipeline") + "' was not specified"))
+			})
+		})
+
+		Context("when resources are returned from the API", func() {
```

Comment:
I can change that, I had replicated it since I'm calling "resources" essentially in `pipeline-overview`, but tackling it directly is probably a more idiomatic approach.